pdf/src/campPrint/picasso/DayColumn.vue

```diff
@@ -54,15 +53,15 @@ export default {
       const radius = '2pt'
       return keyBy(
         this.relevantScheduleEntries.map((scheduleEntry) => {
-          const start = dayjs.utc(scheduleEntry.start)
+          const start = this.$date.utc(scheduleEntry.start)
```

Comment:
Was this change necessary for the translations to work?

Reply:
No, but I stumbled upon it in a global search and found that it's the only place where we use dayjs differently. I thought it would be good to be consistent, if only to make future global searches easier